jenkins/scripts/integration_test.sh

```diff
@@ -122,6 +124,24 @@ fi
 TEST_EXECUTER_IP="$(openstack port show -f json "${TEST_EXECUTER_PORT_NAME}" \
   | jq -r '.fixed_ips[0].ip_address')"
 
+if [ "$OS_REGION_NAME" == "Fra1" ]
+then 
+  # Fetch Free floating IP 
+  FLOATING_IP="$(openstack floating ip list --status DOWN -c "Floating IP Address" -f value | head -1 )"
+
+  if [ -z $FLOATING_IP ]
```

Comment:
Either `[[` is needed or `$FLOATING_IP` should be quoted. I don't think we run `shellcheck` on this script (we probably should?) but shellcheck will fail here.